src/Components/AdvancedSearch/Internal/QueryBuilder/QueryBuilder.styles.ts

```diff
@@ -0,0 +1,182 @@
+            },
+            props.rowCount === 1
+                ? {
+                      gridTemplateColumns: '4fr 4fr 4fr 1fr'
+                  }
+                : {
+                      gridTemplateColumns: '2fr 4fr 4fr 4fr 1fr'
+                  }
+        ],
+        headerText: [
+            classNames.headerText,
+            {
+                fontSize: '16px',
+                margin: 0
+            }
+        ],
+        rowContainer: [
+            classNames.rowContainer,
+            {
+                maxHeight: 150,
+                overflowY: 'auto'
+            }
+        ],
+        subComponentStyles: {
+            searchButton: {
+                root: {
+                    display: 'block'
+                }
+            },
+            addButton: {
+                root: {
+                    display: 'block',
+                    color: props.theme.palette.themePrimary
+                }
+            }
+        }
+    };
+};
+
+/**
+ * Query builder row styles
+ */
+export const rowClassPrefix = 'cb-querybuilderrow';
+const rowClassNames = {
+    root: `${classPrefix}-root`,
+    firstColumn: `${classPrefix}-firstColumn`,
+    inputColumn: `${classPrefix}-inputColumn`,
+    buttonColumn: `${classPrefix}-buttonColumn`
+};
+
+export const getRowStyles = (
+    props: IQueryBuilderRowStyleProps
+): IQueryBuilderRowStyles => {
+    return {
+        root: [
+            rowClassNames.root,
+            {
+                display: 'grid',
+                width: '100%',
+                gridTemplateRows: 'auto',
+                columnGap: '20px',
+                marginTop: 10
+            },
+            props.isOnlyFirstRow
+                ? {
+                      gridTemplateColumns: '4fr 4fr 4fr 1fr'
+                  }
+                : {
+                      gridTemplateColumns: '2fr 4fr 4fr 4fr 1fr'
+                  }
+        ],
+        firstColumn: [rowClassNames.firstColumn],
+        inputColumn: [rowClassNames.inputColumn],
+        buttonColumn: [rowClassNames.buttonColumn],
+        subComponentStyles: {
+            deleteButton: {
+                rootDisabled: {
+                    background: 'transparent'
+                }
+            }
+        }
+    };
+};
+
+export const reactSelectStyles = (isOnlyFirstRow: boolean): StylesConfig => {
+    return {
+        container: (provided) => ({
+            ...provided,
+            fontSize: '14px',
+            padding: '1px'
+        }),
+        control: (provided, state) => ({
+            ...provided,
+            backgroundColor: 'var(--cb-color-bg-canvas)',
+            border: 0,
+            borderRadius: '2px',
+            fontSize: '14px',
+            fontWeight: 400,
+            minHeight: '32px',
+            outline: '1px solid',
+            outlineColor: state.isFocused
+                ? 'var(--cb-color-theme-primary)'
+                : 'var(--cb-color-input-border)'
+        }),
+        indicatorSeparator: () => ({
```

Comment:
the Add new row button needs it's text updated and it's styling fixed to match the designs. It feels off to be floating in from the side like that right above another button which is NOT indented.
Actual
![image](https://user-images.githubusercontent.com/57726991/187562916-1563ac2c-806d-4bbc-8570-91acc2b079fa.png)
Designs 
![image](https://user-images.githubusercontent.com/57726991/187562975-8765182d-3c68-42b0-b2b1-5c1dcc9bc2c4.png)